allenact_plugins/clip_plugin/extra_requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytorch==1.7.1
+torchvision
+ftfy
+regex
+tqdm
+clip @ git+git://github.com/openai/clip@e184f608c5d5e58165682f7c332c3a8b4c1545f2#egg=clip
```

Comment:
If possible it would be better to do something like `@VERSION_NUMBER` or `@BRANCH_NAME` rather than a commit id.

Reply:
Agreed, but the only branch is `main` and there are no releases or tags. Should we prefer this messy commit id over `main` (which is susceptible to changes)?